src/status_im/node/core.cljs

```diff
@@ -60,10 +60,20 @@
     (assoc :ListenAddr ":30304"
            :DataDir (str (:DataDir config) "_dev"))))
 
+(def wakuv2-default-config
```

Comment:
@flexsurfer is there a designated place for this code to live outside of `status_im` folder?
can we migrate this code as part of this work?

Reply:
no we don't, it's better to have a separate PR